mod-source-record-manager-server/src/test/java/org/folio/services/EventDrivenChunkProcessingServiceImplTest.java

```diff
@@ -165,7 +171,12 @@ public void shouldProcessChunkOfRawRecords(TestContext context) {
       List<LoggedRequest> loggedRequests = findAll(postRequestedFor(urlEqualTo(PUBSUB_PUBLISH_URL)));
       context.assertEquals(1, loggedRequests.size());
       Event event = Json.decodeValue(loggedRequests.get(0).getBodyAsString(), Event.class);
-      DataImportEventPayload dataImportEventPayload = Json.decodeValue(event.getEventPayload(), DataImportEventPayload.class);
+      DataImportEventPayload dataImportEventPayload = null;
+      try {
+        dataImportEventPayload = Json.decodeValue(ZIPArchiver.unzip(event.getEventPayload()), DataImportEventPayload.class);
+      } catch (IOException e) {
```

Comment:
why not just throw this exception in the method signature if nothing done when it's caught?

Reply:
You are right

Reply:
@KaterynaSenchenko It is needed because this exception will be in the handler, and there aren`t any ways to handle it in the method level.